android/src/main/java/com/getcapacitor/community/fcm/FCMPlugin.java

```diff
@@ -77,12 +84,12 @@ public void refreshToken(final PluginCall call) {
         FirebaseMessaging.getInstance().deleteToken();
         FirebaseInstallations.getInstance().getToken(false).addOnSuccessListener(getActivity(), instanceIdResult -> {
             JSObject data = new JSObject();
-            data.put("token", instanceIdResult.getToken());
+            data.put("token", tokenResult.getResult());
```

Comment:
@bartacruz is this line really correct?

Reply:
you're expecting `instanceIdResult` there's no `tokenResult` callback on this method